includes/Story_Renderer/Embed.php

```diff
@@ -63,32 +64,53 @@ public function __construct( Story $story ) {
 	 * @return string Rendered block type output.
 	 */
 	public function render( array $args = [] ) {
-		$defaults     = [
+		$defaults = [
 			'align'  => 'none',
 			'height' => 600,
 			'width'  => 360,
 		];
-		$args         = wp_parse_args( $args, $defaults );
-		$align        = sprintf( 'align%s', $args['align'] );
-		$url          = $this->story->get_url();
-		$title        = $this->story->get_title();
-		$poster       = ! empty( $this->story->get_poster_portrait() ) ? esc_url( $this->story->get_poster_portrait() ) : '';
-		$margin       = ( 'center' === $args['align'] ) ? 'auto' : '0';
```

Comment:
If we are changing styling like this, we will have to note this in the change log?

Reply:
This PR itself will be listed in the changelog (as most other PRs), but this markup change is not breaking and does not really warrant a special callout.